docs/make.jl

```diff
@@ -1,18 +1,16 @@
 using SpecialFunctions, Documenter
 
 makedocs(
-    modules = [SpecialFunctions],
-    clean = false,
-    format = :html,
-    sitename = "SpecialFunctions.jl",
-    authors = "Jeff Bezanson, Stefan Karpinski, Viral B. Shah, et al.",
-    pages = [
-        "Home" => "index.md",
+    modules     = [SpecialFunctions],
+    sitename    = "SpecialFunctions.jl",
+    authors     = "Jeff Bezanson, Stefan Karpinski, Viral B. Shah, et al.",
+    pages       = [
+        "Home"      => "index.md",
         "Functions" => "special.md",
     ],
 )
 
 deploydocs(
-    repo = "github.com/JuliaMath/SpecialFunctions.jl.git",
-    target = "build",
-)
+    repo    = "github.com/JuliaMath/SpecialFunctions.jl.git",
+    target  = "build",
+)
```

Comment:
Please, leave the newline at the end of the line.

Reply:
Ok, just updated my editor's settings.

Reply:
appended newline to file in new commit